doc/dev-reference/getting_started.rst

```diff
@@ -79,22 +81,23 @@ These can be determined at library compile time, see :ref:`target_arch`.
 Software
 ========
 
-As a software library, Hyperscan doesn't impose any particular runtime
-software requirements, however to build the Hyperscan library we require a
-modern C and C++ compiler -- in particular, Hyperscan requires C99 and C++11
+As a software library, Vectorscan doesn't impose any particular runtime
+software requirements, however to build the Vectorscan library we require a
+modern C and C++ compiler -- in particular, Vectorscan requires C99 and C++11
 compiler support. The supported compilers are:
 
     * GCC, v4.8.1 or higher
```

Comment:
GCC v9 and Clang v5 are now the default minimum as C++17 is required.

Reply:
Sure